src/models.py

```diff
@@ -463,6 +463,9 @@ def build_task_specific_modules(
         d_sent = args.d_hid + (args.skip_embs * d_emb)
         hid2voc = build_lm(task, d_sent, args)
         setattr(model, '%s_hid2voc' % task.name, hid2voc)
+    elif isinstance(task, SpanTask):
+        module = SpanClassifierModule(task, d_sent, task_params, num_spans=task.num_spans)
```

Comment:
Is there a reason we don't have a build_span_classifier here but do have build_lm?